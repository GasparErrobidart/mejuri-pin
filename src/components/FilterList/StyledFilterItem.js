import styled      from 'styled-components'

const StyledFilterItem = styled.div`
  background: white;
  padding: 15px 0;
  border-top:1px solid ${({theme})=> theme.colors.lightGray};
  cursor:pointer;
  @media (min-width:768px){
    background:none;
  }
`

export default StyledFilterItem
