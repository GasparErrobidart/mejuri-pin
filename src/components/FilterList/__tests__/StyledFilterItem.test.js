import '@testing-library/jest-dom'
import 'jest-styled-components'

import React                 from 'react'
import {ThemeProvider}       from 'styled-components'
import { cleanup }           from '@testing-library/react'
import renderer              from 'react-test-renderer'

import theme                 from '../../GlobalStyle/themes/mejuri-pin'

import StyledFilterItem   from '../StyledFilterItem'


afterEach(cleanup)

test('Renders "StyledFilterItem" component', () => {
  const testText        = "StyledFilterItem Test Text"
  const component       = renderer.create(
    <ThemeProvider theme={theme}>
      <StyledFilterItem>{testText}</StyledFilterItem>
    </ThemeProvider>);
  const tree            = component.toJSON()
  expect(tree).toMatchSnapshot();
});
