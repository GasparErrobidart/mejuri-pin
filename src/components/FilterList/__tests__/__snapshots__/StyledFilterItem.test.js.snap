// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders "StyledFilterItem" component 1`] = `
.c0 {
  background: white;
  padding: 15px 0;
  border-top: 1px solid #F3F3F3;
}

@media (min-width:768px) {
  .c0 {
    background: none;
  }
}

<div
  className="c0"
>
  StyledFilterItem Test Text
</div>
`;
